README.md

```diff
@@ -49,25 +49,45 @@ This repository provides a hermetic [Strawberry Perl](https://strawberryperl.com
 
 ## Using Perl Modules
 
-This is the first stab at getting a more mature set of Perl rules for Bazel.  Currently, it is a manual process and, hopefully, it will be a map for automation later on.
+Perl modules from [CPAN](https://www.cpan.org/) can be generated using the `cpan_compiler` rule in
+conjunction with the `cpan` module extension.
 
 ### Current Steps
 
-* Manually download the module that you want to use.
-* Add the actual files that you need to your repository.
-  * Highly recommended that you place the files in the directory structure that each Perl file is unpacked into (you may need to run `perl Makefile.PL; make` to see the final paths)
-  * Recommended to create a 'cpan' directory and place the files (in their required path) there.
-  * Test::Mock::Simple does **NOT** follow this pattern as it is being used as a practical example - please see 'Simple Pure Perl Example' section.
-* Add the new module's information to the BUILD file in the root directory of all your modules.
-  * the target in the `deps` attribute
-    * At this time compiled files (result of XS) will be put in the `srcs` attribute
-  * the directory where the module lives in the `env` attribute for the `PERL5LIB` variable
-
-#### Dependencies
-
-The process needs to be repeated for any dependencies that the module needs.
-
-Eventually, this should be an automated process.
+1. Create a `cpanfile` per the [Carton](https://metacpan.org/pod/Carton) documentation.
+2. Create an empty `*.json` will need to be created for Bazel to use a lockfile (e.g. `cpanfile.snapshot.lock.json`)
+3. Define a `cpan_compiler` target:
+
+  ```python
```

Comment:
Nit: Not a big difference but GitHub-flavored Markdown supports Starlark highlighting  (https://github.com/github-linguist/linguist/blob/main/vendor/README.md)
```suggestion
  ```Starlark
```

Reply:
`python` is already used in other places so I'd rather just be consistent with this change.